readthedocs/projects/tasks.py

```diff
@@ -573,6 +577,16 @@ def run_build(self, docker, record):
 
         if self.build_env.failed:
             self.send_notifications(self.version.pk, self.build['id'])
+            # send build failure status to git Status API
+            self.send_build_status(
+                self.build['id'], BUILD_STATUS_FAILURE
+            )
+
+        if self.build_env.successful:
```

Comment:
Should this check for the version being external before sending, same as above?

Reply:
@ericholscher I'm checking that inside the task
https://github.com/readthedocs/readthedocs.org/blob/0011043e85c3499c66079893806b06d1065aea84/readthedocs/projects/tasks.py#L1807-L1808

We are using this in few places so it did not look good to check everywhere so I put it here.